File: C/Lista_03(Estrutura_Repetitiva)/Fatorial.c
#include <stdio.h>

int main () {

    int n;

    printf("Digite o valor de N: ");
    scanf("%d", &n);

    int fat = 1;

    for(int i=n;i>=1;i--){
        fat = fat * i;
    }

    if(n==0){
        fat = 1;
    }

    printf("FATORIAL = %d", fat);

    return 0;
    
}

File: Java/Lista_03(Estrutura_Repetitiva)/Fatorial.java
import java.util.Scanner;

public class Fatorial {

	public static void main(String[] args) {
		
		Scanner s = new Scanner(System.in);
		
		System.out.print("Digite o valor de N: ");
		int n = s.nextInt();
		
		int fat = 1;
		
		if(n==0 || n==1) {
			System.out.println("FATORIAL = 1");
		}
		else {
			for(int i=1;i<=n;i++) {
				fat*=i;
			}
			System.out.println("FATORIAL = " + fat);
		}
		
		s.close();

	}

}
